repos/c2sm/packages/fdb/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fdb(SpackFdb):
+    """FDB (Fields DataBase) is a domain-specific object store developed at
+    ECMWF for storing, indexing and retrieving GRIB data."""
+
```

Comment:
```suggestion

    version("5.11.23", sha256="09b1d93f2b71d70c7b69472dfbd45a7da0257211f5505b5fcaf55bfc28ca6c65")
```

Reply:
Shall we add 5.11.23 as well?

Reply:
If yes, we should add system tests for it.

Reply:
Added in https://github.com/C2SM/spack-c2sm/pull/848/commits/93981bc51c0db8e56c80fccb6bdda8c1347aa8a6